ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -868,6 +857,27 @@ class JITModBuilder {
     }
   }
 
+  GlobalVariable* cloneGlobalVariable(Value *V) {
+    // If there's a reference to a GlobalVariable, copy it over to the
+    // new module.
+    GlobalVariable *OldGV = cast<GlobalVariable>(V);
+    // Global variable is a constant so just copy it into the trace.
```

Comment:
Is it the address of a global that is constant? You have to `load` from them, don't you? Maybe clarify?

Reply:
This comment is just copied over from the old code. I'm actually not sure what this is supposed to say.

Reply:
I suggest we kill it?

Reply:
Done [dc3e642](https://github.com/ykjit/yk/pull/563/commits/dc3e6426c08275b870c44757a1c69e80b52cc497)